.github/workflows/build.yml

```diff
@@ -22,6 +22,8 @@ jobs:
             redmine_version: 4.2-stable
           - ruby_version: "3.2"
             redmine_version: 5.0-stable
+          - ruby_version: "2.7"
+            redmine_version: master
```

Comment:
The current Redmine master can use Ruby between 3.0 and 3.4.
https://github.com/redmine/redmine/blob/c3fe22476231adff5f6fef2e1692be8024dd0c44/Gemfile#L3

Reply:
How about adding it to the `exclude`?

https://github.com/otegami/redmine_wiki_extensions/blob/develop/.github/workflows/build.yml#L16-L24